checker/examples/Makefile

```diff
@@ -4,7 +4,6 @@ all: compile
 
 compile:
 	$(JAVAC) *.java
-	$(JAVAC) units-extension/*.java
```

Comment:
There's a separate ant target (units-extension-tests) which executes the make command in units-extension directory, in turn compiling and executing the updated units-extension example project.